readthedocs/gold/templates/gold/projects.html

```diff
@@ -11,6 +11,13 @@
 
 {% block edit_content %}
 
+  <p class="empty">
+    {% blocktrans trimmed %}
+      <strong>Note:</strong> this is a legacy feature.
+      New gold members cannot sponsor projects to be ad-free.
```

Comment:
It doesn't feel good to me to expose this to new gold users.

I'm not sure what's best here, but maybe removing this page completely for new gold users is better, instead of showing them that they could have sponsored their projects.

Reply:
I'm on the fence about whether to include it, but I do think we should probably prompt them to email us here about removing ads if they want as well. Or perhaps some other messaging?

Reply:
> It doesn't feel good to me to expose this to new gold users.

The page is removed for new gold users. New users will be redirected. This message is shown only to older gold users to let them know that we no longer do this.

Reply:
Excellent! :100: